sources/packages/load-test/src/utils/auth/models.ts

```diff
@@ -6,6 +6,13 @@ export interface UserPasswordCredential {
   password: string;
 }
 
+/**
+ * Client secret authentication credential.
+ */
+export interface ClientSecretCredential {
+  clientSecret: string;
```

Comment:
As discussed, let have this as a AC for the next story, to create secrets for the client